android/src/main/java/nl/fhict/happynews/android/PostManager.java

```diff
@@ -32,19 +34,22 @@ private PostManager() {
      * @param c context of the apps
      * @return
      */
-    public ArrayList<Post> getNewPosts(Context c){
-        final ArrayList<Post> newPosts = new ArrayList<>();
+    public void updatePosts(Context c) {
+        newPosts = new ArrayList<>();
```

Comment:
What is the use of this variable? The posts are added to this list, but are never queried.